test/e2e/test_02_server.py

```diff
@@ -77,36 +77,36 @@ def test_lausanne_thumbnail_alpha(self, manager):
         manager.expect_status_code("/thumbs/{}.jpg/full/full/0/default.jpg".format(filename), 200)
 
     def test_concurrency(self, manager):
-        """handle many concurrent requests for different URLs"""
+        """handle many concurrent requests for different URLs (this may take a while, please be patient)"""
 
         # The command-line arguments we want to pass to ab for each process.
         
         filename = "load_test.jpx"
 
         ab_processes = [
```

Comment:
@lrosenth @benjamingeer The tests pass on my MacBook Air (2.2 GHz Intel Core i7, 8 GB 1600 MHz DDR3, macOs Sierra).

The load test's settings may have to be adapted for production conditions. Maybe we could define different scenarios. For now, it is important that bugs like the non thread safe XMP lib are detected.